test/test_xmath.cpp

```diff
@@ -19,6 +19,12 @@ namespace xt
      * Basic operations
      **********************/
 
+    TEST(xmath, pow_add_pow)
+    {
+        shape_type shape = {3, 2};
+        xarray<double> a(shape, 4.5);
+        EXPECT_EQ((pow(a, 2.f) + pow(a, 2.f))(0, 0), std::pow(a(0, 0), 2.f) + std::pow(a(0, 0), 2.f));
+    }
```

Comment:
I added this test because I observed a problem with the missing `operator[](xindex)` on xscalars. But actually, this test didn't trigger the bug, so this is kind of an artifact.